mrjob/emr.py

```diff
@@ -2454,7 +2450,14 @@ def _pool_hash(self):
 
         if self._opts['bootstrap_mrjob']:
             things_to_hash.append(mrjob.__version__)
-        return hash_object(things_to_hash)
+
+        things_json = json.dumps(things_to_hash, sort_keys=True)
+        if not isinstance(things_json, bytes):
+            things_json = things_json.encode('utf_8')
```

Comment:
Profiling shows that 'utf-8' can be twice as fast as 'UTF-8' or 'utf_8' in json-heavy applications.